### PR TITLE
Poppler: bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: app-text/poppler/poppler-20.11.0.recipe [hunk 1]
```diff
@@ -273,15 +278,15 @@ REQUIRES="
 PROVIDES_devel="
 	poppler${secondaryArchSuffix}_devel = $portVersion
 	devel:libpoppler$secondaryArchSuffix = $libVersionCompat
-	devel:libpoppler${secondaryArchSuffix}_cpp = $libCppVersionCompat
-	devel:libpoppler${secondaryArchSuffix}_glib = $libGlibVersionCompat
-	devel:libpoppler${secondaryArchSuffix}_qt5 = $libQt5VersionCompat
+	devel:libpoppler_cpp$secondaryArchSuffix = $libCppVersionCompat
+	devel:libpoppler_glib$secondaryArchSuffix = $libGlibVersionCompat
+	devel:libpoppler_qt5$secondaryArchSuffix = $libQt5VersionCompat
```
Comment (korli) [inline app-text/poppler/poppler-20.11.0.recipe:283]: Why is it modified?

File: app-text/poppler/poppler-20.11.0.recipe [hunk 2]
```diff
@@ -264,6 +267,8 @@ REQUIRES="
 	lib:libintl$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:liblcms2$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libnss3$secondaryArchSuffix
```
Comment (korli) [inline app-text/poppler/poppler-20.11.0.recipe:271]: but devel:libnss3$secondaryArchSuffix is commented out

Comment (Begasus) [reply app-text/poppler/poppler-20.11.0.recipe:271]: It's mentioned in line 385 (BUILD_REQUIRES)

**Comment by Begasus** [comment]
Builds ok on 32 bit

File: app-text/poppler/poppler-20.11.0.recipe [hunk 3]
```diff
@@ -294,6 +299,7 @@ REQUIRES_devel="
 	devel:libjpeg$secondaryArchSuffix
 	devel:liblcms2$secondaryArchSuffix
 	devel:libopenjp2$secondaryArchSuffix
+#	devel:libnss3$secondaryArchSuffix
```
Comment (korli) [inline app-text/poppler/poppler-20.11.0.recipe:302]: Why add this Line?

Comment (Begasus) [reply app-text/poppler/poppler-20.11.0.recipe:302]: That's one for @extrowerk :)

**Comment by Begasus** [comment]
Seems to be OK, just did a build for TexStudio with this one, if this is merged, TexStudio should get a revbump :) (first merge poppler_data)

File: app-text/poppler/poppler-20.11.0.recipe (hunk 3 above)
Comment (extrowerk) [reply app-text/poppler/poppler-20.11.0.recipe:302]: AFAIR poppler could use libnss, the configure stage searching for it, but was unable to find it. Could be removed.

**Comment by Begasus** [comment]
Looking at the packages I see that for instance: libpoppler-glib.so.8.19.0 (and .8) is provided in the base package as well as in the *_glib package, is that correct?
Didn't check, but shouldn't those be separated from the base package?

**Comment by extrowerk** [comment]
>Looking at the packages I see that for instance: libpoppler-glib.so.8.19.0 (and .8) is provided in the base package as well as in the *_glib package, is that correct?
Thats doesn't seems to be ok.

Basically the base package provides the cli tools and the poppler (which is the C version) and the cpp version.
qt5 provides the qt5 lib, while glib the glib one.

I checked on my side again, the base package contains only the c-version and the cpp version lib. Please check again on your side, i think you made a mistake somewhere.

**Pull request merged**